db/migrate/20210519155027_add_article_id_to_rating.rb

```diff
@@ -0,0 +1,5 @@
+class AddArticleIdToRating < ActiveRecord::Migration[6.1]
+  def change
+    add_column :ratings, :article_id, :integer
+  end
+end
```

Comment:
Are you able to access Rating from the article based on this? Shouldn't it have foreign_key?

Reply:
We forgot about that thing. We manually added user_id and article_id to Rating to make the association bi-directional. So it works, just not the most elegant implementation